frontend/src/components/search/SearchBar.js

```diff
@@ -71,51 +72,44 @@ export class SearchBar extends Component {
     });
   };
 
-  render = () => (
-    <Col
-      id="searchBox"
-      md={4}
-      lg={6}
-      xl={8}
-      style={{
-        marginLeft: "auto",
-        marginTop: "auto",
-        marginBottom: "auto",
-        paddingRight: "20px",
-      }}
-    >
-      <Dropdown
-        overlay={
-          <Card>
-            <SearchBarDropdown
-              history={this.props.history}
-              searchTerm={this.state.searchTerm}
-              searchTermHistory={this.state.searchTermHistory}
-              searchBarResults={this.props.searchBarResults}
-            />
-          </Card>
-        }
-        getPopupContainer={() => document.getElementById("searchBox")}
-        trigger={[""]}
-        visible={this.state.isSelected}
-      >
-        <Search
-          size="large"
-          value={this.state.searchTerm}
-          placeholder={this.state.isSelected ? selectedPlaceholderText : defaultPlaceholderText}
-          onSearch={(searchTerm) => this.search(searchTerm)}
-          onChange={this.changeSearchTerm}
-          onClick={this.clearSearchBar}
-          onFocus={this.clearSearchBar}
-          onBlur={() =>
-            this.setState({
-              isSelected: false,
-            })
+  render() {
+    // The default behaviour will fix the dropdown to the parent (searchBox), except in the navBar where the user can scroll beyond the searchBox.
```

Comment:
This comment may be useful if I were more familiar with this component, but from what I can see here, I don't understand it

Reply:
ah, I was wondering how deep to go with this comment. Not deep enough apparently. I'll remove it